app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,88 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.domain.ChangeTaskRequest;
+import com.codesoom.assignment.domain.Task;
+import com.codesoom.assignment.service.TaskService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private final TaskService taskService;
+
+    @Autowired
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    /**
+     * 입력 받은 숫자 타입의 taskId와 같은 id를 가진 작업을 조회해 리턴한다.
+     *
+     * @param taskId 입력 받은 숫자 타입 taskId
+     * @return 입력 받은 taskId와 같은 id를 가진 작업 리턴, 존재하지 않으면 예외 메시지를 던짐
+     * @throws ResponseStatusException 찾는 작업이 존재하지 않으면 던집니다.
+     */
```

Comment:
다음 JavaDoc 주석과 비교해 보세요. 아래는 제가 방금 작성한 것입니다. 어떤 점이 다른가요?

```suggestion
    /**
     * 주어진 식별자를 갖는 작업을 찾아 리턴한다.
     *
     * @param taskId 작업의 식별자
     * @return 작업
     * @throws ResponseStatusException 작업을 찾지 못한 경우
     */
```

Reply:
주석을 보고 짧은 시간 안에 해당 메서드를 확인할 수 있었으며 파라미터, 객체의 변경에 닫혀있습니다.